lib/figures.js

```diff
@@ -1,8 +1,8 @@
-import { blue, redBright, yellow } from 'colorette'
+import chalk from 'chalk'
 import { figures } from 'listr2'
 
-export const info = blue(figures.arrowRight)
+export const info = chalk.blue(figures.arrowRight)
 
-export const error = redBright(figures.cross)
+export const error = chalk.redBright(figures.cross)
 
-export const warning = yellow(figures.warning)
+export const warning = chalk.yellow(figures.warning)
```

Comment:
Chalk uses `supports-color` internally as well, so this makes sure our own colored text doesn't mess up terminals where the previous implementation failed to detect support correctly.